File: javascript/8kyu/No zeros for heros.js
// Numbers ending with zeros are boring.

// They might be fun in your world, but not here.

// Get rid of them. Only the ending ones.

// 1450 -> 145
// 960000 -> 96
// 1050 -> 105
// -1050 -> -105

// https://www.codewars.com/kata/570a6a46455d08ff8d001002

function noBoringZeros(n) {
    while(n%10==0 && n!=0){
        n/=10;
    }
    return n;
}

File: java/8kyu/No zeros for heros.java
// Get rid of them. Only the ending ones.

// https://www.codewars.com/kata/570a6a46455d08ff8d001002

public class NoBoring {
    public static int noBoringZeros(int n) {
        if (n == 0)
            return n;
            
        while (n % 10 == 0)
            n /= 10;       
        
        return n;
    }
}